src/annotations/AnnotationManager.php

```diff
@@ -299,6 +306,22 @@ protected function getAnnotations($class_name, $member_type = self::MEMBER_CLASS
                 }
             }
 
+            if ($this->_traitsSupported && $member_name) {
```

Comment:
Sorry—was confused about what you meant. As I understand it, getAnnotations, when given a method/property filter, calls itself with `MEMBER_CLASS` in order to check whether the class is annotated with `@Stop`. That recursive call doesn't actually load the annotation for `$member_name` unless I'm missing something. A method/property annotation will only be cached when `getAnnotation` is called for that method/property specifically.